Form/Type/SearchType.php

```diff
@@ -17,7 +19,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         $builder
             ->add(
                 'term',
-                'text',
+                TextType::class,
```

Comment:
ja, maar dan moet ook overal php minimum versie naar 5.5 gebumped worden (dus ook in composer.json en readme files enzo)